gsoc/2021/mentors.md

```diff
@@ -3,12 +3,14 @@ title: Summary of Mentors GSoC 2021
 layout: plain
 ---
 
-## Full Mentor List (Name, Email, Org)
+**Note for contributors:** entries must be sorted in last name alphabetic order
```

Comment:
Shouldn't this go in its own commit PR?

Reply:
Yes in principle, but given that it is a minor detail changing the same file, I would let it like this for convenience.